Add Default Machine Setting (#383)
gamer-project/gamer: doc/wiki/Installation.md

   Please see [[Machine Configuration File | Installation:-Machine-Configuration-File]].

2. Set your machine configuration file as default

   ```bash
   sh tool/config/set_settings.sh --local --machine=your_machine
   ```

   > [!NOTE]
   > If you want to set the default machine configuration file for all of the GAMER copies under your user account, use the `--global` option instead of `--local`.
   Still, you can override the global setting for the individual GAMER copies with the `--local` option.

>    Furthermore, you can override the default setting with the `--machine=` [[option | Installation:-Option-List]] in `configure.py`.

Reviewer: ```suggestion
   Furthermore, you can override the default setting by passing the [[--machine | Installation:-Option-List#--machine]]=`your_machine` when executing `configure.py`. 
```

Author: @ChunYen-Chen Is there a section of `--machine` in `Installation:-Option-List`? I'm wondering where will `Installation:-Option-List#--machine` link to?

Reviewer: > Is there a section of --machine in Installation:-Option-List?

No. Can you add the following code to the `--machine` section? This will add the link to `--machine`.
```markdown
<a name="--machine"></a>
```

Author: Added. I put it under `## configure.py Only` to not break the table.
